lexicons/com/atproto/admin/getSubjectState.json

```diff
@@ -0,0 +1,39 @@
+{
+  "lexicon": 1,
+  "id": "com.atproto.admin.getSubjectState",
+  "defs": {
+    "main": {
+      "type": "query",
+      "description": "Fetch the service-specific the admin state of a subject (account, record, or blob)",
+      "parameters": {
+        "type": "params",
+        "properties": {
+          "did": { "type": "string", "format": "did" },
+          "uri": { "type": "string", "format": "at-uri" },
+          "blob": { "type": "string", "format": "cid" }
```

Comment:
If you want the status for a blob do you end-up passing `(did, blob)`?

Reply:
yeah that's what i was thinking